csrankings-a.csv

```diff
@@ -207,6 +207,7 @@ Adam Bates 0001,Univ. of Illinois at Urbana-Champaign,http://adambates.org,1dVgl
 Adam Belay,Massachusetts Institute of Technology,https://engineering.mit.edu/faculty-research/year-2017,AKQec5wAAAAJ
 Adam Belloum,University of Amsterdam,https://www.uva.nl/profiel/b/e/a.s.z.belloum/a.s.z.belloum.html,tzFsIF4AAAAJ
 Adam Breuer,Dartmouth College,https://www.adambreuer.com/,1b3CsE4AAAAJ
+Adam Charles,Johns Hopkins University,https://www.bme.jhu.edu/ascharles/,c8RKLp0AAAAJ

```

Comment:
<img width="1217" alt="Screen Shot 2024-11-04 at 10 41 45 AM" src="https://github.com/user-attachments/assets/23f9ce35-eae2-4bee-b0fe-495ade0e6ad6">

Reply:
From here: https://www.cs.jhu.edu/people/cs-affiliates/joint-and-secondary-faculty/